x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/available_sources_list.tsx

```diff
@@ -29,12 +30,10 @@ interface AvailableSourcesListProps {
 }
 
 export const AvailableSourcesList: React.FC<AvailableSourcesListProps> = ({ sources }) => {
-  const {
-    fpAccount: { minimumPlatinumLicense },
-  } = useValues(AppLogic);
+  const { hasPlatinumLicense } = useValues(LicensingLogic);
```

Comment:
Are we sure that it has the same meaning as minimumPlatinumLicense?

Reply:
Yep! https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/licensing/licensing_logic.ts#L47

Reply:
Nice, thanks for double checking!